traptor/traptor.py

```diff
@@ -155,6 +157,19 @@ def _setup_kafka(self):
             self.logger.debug('Kafka_enabled setting: {}'.format(self.kafka_enabled))
             self.kafka_conn = None
 
+    def _gen_kafka_success(self):
+        def kafka_success(tweet, response):
+            self.logger.info("Tweet sent to kafka: {}".format(tweet.get('id_str', None)))
+            dd_monitoring.increment('tweet_to_kafka_success')
+        return kafka_success
+
+    def _gen_kafka_failure(self):
+        def kafka_failure(e):
+            self.logger.error("KAFKA FAILURE: {}".format(e))
+            self.logger.error("Unable to add tweet to Kafka: {}".format(traceback.format_exc()))
```

Comment:
Both tracebacks here can be condensed probably into a single exception message with the tracebacks inside the `extras` dict as `ex` or `ex_kafka`